cameraview/src/main/java/com/otaliastudios/cameraview/CameraView.java

```diff
@@ -1828,6 +1932,41 @@ public void run() {
         }
     }
 
+    public static class OverlayLayoutParams extends FrameLayout.LayoutParams {
```

Comment:
As said below, this class should always have `isOverlay=true` (so you can actually remove this boolean from here). 

So this will be empty for now but one day we will add things like `drawInPreview`, `drawInPictureSnapshot` and `drawInVideoSnapshot` which will be held by this class and checked by the `OverlayLayout`.